contrib/vagrant/provision-util.sh

```diff
@@ -56,7 +57,8 @@ os::provision::base-install() {
 os::provision::install-cmds() {
   local deployed_root=$1
 
-  cp ${deployed_root}/_output/local/bin/linux/amd64/{openshift,oc,osadm} /usr/bin
+  local output_path=$(os::provision::get-bin-output-path "${deployed_root}")
```

Comment:
The output of substitutions `$()` should be quoted to ensure word-breaking doesn't do anything we don't want it to: `local output_path="$( os::provision::get-bin-output-path "${deployed_root}" )"